sdcm/kcl_thread.py

```diff
@@ -125,20 +125,19 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
 
             while not self._stop_event.is_set():
                 node: BaseNode = self.db_node_to_query(loader)
+                node.running_nemesis = "Compare tables size by cf-stats"
                 node.run_nodetool('flush')
 
-                src_size = node.get_cfstats(src_table)['Number of partitions (estimate)']
                 dst_size = node.get_cfstats(dst_table)['Number of partitions (estimate)']
+                src_size = node.get_cfstats(src_table)['Number of partitions (estimate)']
 
+                node.running_nemesis = None
                 status = f"== CompareTablesSizesThread: dst table/src table number of partitions: {dst_size}/{src_size} =="
                 LOGGER.info(status)
                 InfoEvent(status)
 
                 if src_size == 0:
                     continue
-                if dst_size >= src_size:
```

Comment:
without some kind of stopping logic, this would run forever

Reply:
added timeout to break out of loop.